### ISSUE TITLE
Link to goerep when letter

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#549]

**Comment by dietmarpravida** [comment]
d.h. man müsste in excel eine Spalte ergänzen, wo die Briefnummer einzutragen ist?

**Comment by gerritbruening** [comment]
Ja, genau (falls es die Spalte noch nicht gibt; evtl. gibt es WA-Briefnummer, aber ich würde dann auf die Briefnummer nach dem Goerep umstellen, weil die eindeutig auch über alle Bände hinweg ist).

**Comment by thvitt** [comment]
bevor man das macht sollte man mal mit den goerep-leuten klären, auf was für eine Nummer man da eigentlich Links setzen kann, wenn man einen Einzelbrief verlinken will. In den Links stehen IDs wie in https://ora-web.swkk.de/goe_rep_online/repertorium.Vollanzeige1?d_id=10439, die in den 'gedruckten' Daten nirgends auftauchen, und auch Suchen über die Suchmaske nach sowas wie WA-Nr. `00047` oder `GB 1, Nr. 1` trunkiert automatisch und liefert entsprechend mehrere Ergebnisse.

**Comment by dietmarpravida** [comment]
wir müssen wohl klären, wie wir am einfachsten an die IDs kommen; sie alle aus den Adressen herauszuziehen, wäre unangenehm

**Comment by dietmarpravida** [comment]
im Augenblick haben wir, glaube ich, noch gar nichts

**Comment by thvitt** [comment]
dochdoch, kind of, wahlloser exczerpt aus testimony/wa_IV/wa_IV_10.xml:

```xml
            <div type="letter">
                <head>
                    <num type="wa-volume">10</num>/<num type="wa-letter">3102</num>. <lb/>
                    <lb/>An <persName>Friedrich Schiller</persName>
                    <date when="1794-12-02"/>
                </head>
```

einzelne Einträge in der Metadatentabelle haben auch ein entsprechendes Feld, aber das sind nur ne Handvoll.

**Comment by dietmarpravida** [comment]
ah, okay, dann spreche ich nochmal in Weimar vor, damit sie uns eine Konkordanz verschaffen

**Comment by dietmarpravida** [comment]
[Konkordanz_Ident_WA.xlsx](https://github.com/faustedition/faust-gen-html/files/2434052/Konkordanz_Ident_WA.xlsx)

**Comment by dietmarpravida** [comment]
Wo sollen die ID's hin? In die excel-Tabelle?

**Comment by dietmarpravida** [comment]
@thvitt soll das noch in 1.0?

**Comment by dietmarpravida** [comment]
@thvitt das ist geklärt oder? Wenn ja, bitte schließen.

**Comment by thvitt** [comment]
Wir haben noch keine Links, oder?

D.h. der nächste Schritt ist (wohl bei mir) mithilfe der Konkordanz die Briefe zu verlinken (und zu prüfen, ob das überall klappt)

**Comment by thvitt** [comment]
Die neue URL ist wohl [`https://ores.klassik-stiftung.de/ords/f?p=402:2:2595842258813::NO:RP:P2_ID,P2_QUELLE,P2_ANSICHT:{{Ident}},210,1`](https://ores.klassik-stiftung.de/ords/f?p=402:2:2595842258813::NO:RP:P2_ID,P2_QUELLE,P2_ANSICHT:53774,210,1)
